nexus/auth/src/authz/omicron.polar

```diff
@@ -440,6 +440,27 @@ resource ConsoleSessionList {
 has_relation(fleet: Fleet, "parent_fleet", collection: ConsoleSessionList)
 	if collection.fleet = fleet;
 
+# Allow silo admins to delete user sessions and list user tokens
+resource SiloUserAuthnList {
+    permissions = [ "modify", "list_children" ];
+    relations = { parent_silo: Silo };
+
+    # A silo admin can modify (e.g., delete) a user's sessions.
+    "modify" if "admin" on "parent_silo";
+
+    # A silo admin can list a user's tokens and sessions.
+    "list_children" if "admin" on "parent_silo";
```

Comment:
This is a little funny because I'm using `list_children` to determine whether you can list sessions and tokens (only self and silo admin can) but then I'm using `modify` on the list itself to determine whether you can do the logout delete-all operation. It works fine, but it goes slightly against the grain of how I know it's _supposed_ to work. We just don't have many delete all type things.

Reply:
Thinking more about it, I think using an individual `ConsoleSession` and `ConsoleToken` authz resource here couldn't make sense because you'd have to instantiate one for every session and token you're deleting. So I think the list approach is correct. _Maybe_ to make it more strictly correct we could introduce a new permission `delete_children` so we're not overloading `modify`, but for this one case it's probably overkill. Plus, the semantics of `modify` are not really set in stone and already imply deletion, so to make it make sense you can just think of deleting the list of sessions as deleting all the sessions.